python/ray/serve/_private/build_app.py

```diff
@@ -45,6 +46,8 @@ def __contains__(self, key: object):
 class BuiltApplication:
     # Name of the application.
     name: str
+    route_prefix: Optional[str]
+    logging_config: Optional[LoggingConfig]
```

Comment:
These are now part of the `BuiltApplication` to make it easier to group things up without adding an extra intermediate data structure (that would presumably have a `BuiltApplication` + `route_prefix` + `logging_config`).